src/components/graph/Graph.tsx

```diff
@@ -255,17 +249,12 @@ export const Graph: FC<IGraphProps> = (props: IGraphProps) => {
   useEffect(() => {
     const zoomSelection: Selection = d3.select(`#${graphContainerId}`);
     zoomSelection.call(zoomRef.current);
+    if (!graphConfig.zoom.zoomByScroll) {
```

Comment:
By defalut, this value is not set (undefined), then zoom by scoll will be disabled. 
I think it is good since scroll to zoom seems a little bit annoying to me. But just want to double confirm with you if this is what you want.

Reply:
By default this value is set to true.